File: VisitBosnia/VisitBosnia.Services/Database/City.cs
using System;
using System.Collections.Generic;

namespace VisitBosnia.Services.Database
{
    public partial class City
    {
        public City()
        {
            Agencies = new HashSet<Agency>();
            Forums = new HashSet<Forum>();
            TouristFacilities = new HashSet<TouristFacility>();
        }

        public int Id { get; set; }
        public string Name { get; set; } = null!;
        public string County { get; set; } = null!;
        public string ZipCode { get; set; } = null!;
        public byte[]? Image { get; set; }

        public virtual ICollection<Agency> Agencies { get; set; }
        public virtual ICollection<Forum> Forums { get; set; }
        public virtual ICollection<TouristFacility> TouristFacilities { get; set; }
    }
}

File: VisitBosnia/visit_bosnia_mobile/lib/model/city/city.dart
class City {
  int? id;
  String? name;
  String? county;
  String? zipCode;

  City({this.id, this.name, this.county, this.zipCode});

  City.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    name = json['name'];
    county = json['county'];
    zipCode = json['zipCode'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['id'] = this.id;
    data['name'] = this.name;
    data['county'] = this.county;
    data['zipCode'] = this.zipCode;
    return data;
  }
}
